### DISCUSSION TITLE
Filter variants with criteria : LOF CONSTR ?

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#4443] [Q&A] [answered]
Hello,

I would like to know if it's possible to filter the variants (in Project > Variant Search) using the criteria "LOF CONSTR" ?

Thank you in advance for your answer.

**Comment by hanars** [comment, marked as answer]
You can filter in seqr using any of the filters available in the user interface, which include VEP predicted consequences and in silico scores. If there is a not a filter in the user interface that allows you to filter on something, there is no way to do it
